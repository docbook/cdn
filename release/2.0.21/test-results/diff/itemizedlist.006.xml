<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Unit Test: itemizedlist.006</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Unit Test: itemizedlist.006</h2>
            <div class="author">
               <h3>
                  <span class="personname">
                     <span class="firstname">Adam</span>
                     <span class="surname">Di Carlo</span>
                  </span>
               </h3>
            </div>
         </header>
         <p deltaxml:deltaV2="A">Itemized list mark styles.</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">itemizedlist</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">content</deltaxml:attributeValue></dxa:class></deltaxml:attributes>
            <p deltaxml:deltaV2="B">Itemized list mark styles.</p><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">title</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">itemizedlist</deltaxml:attributeValue></dxa:class></deltaxml:attributes><deltaxml:textGroup deltaxml:deltaV2="A!=B"><deltaxml:text deltaxml:deltaV2="A">Bullet marks</deltaxml:text><deltaxml:text deltaxml:deltaV2="B">
               </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" class="title">Bullet marks</div><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
               </deltaxml:text></deltaxml:textGroup><ul deltaxml:deltaV2="B">
                  <li style="list-style-type: disc">
                     <p>One</p>
                  </li>
                  <li style="list-style-type: disc">
                     <p>Type</p>
                  </li>
               </ul><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup></div>
            <ul deltaxml:deltaV2="A">
               <li style="list-style-type: disc">
                  <p>One</p>
               </li>
               <li style="list-style-type: disc">
                  <p>Type</p>
               </li>
            </ul><div deltaxml:deltaV2="B" class="itemizedlist">
               <div class="title">Box marks</div>
               <ul>
                  <li style="list-style-type: square">
                     <p>One</p>
                  </li>
                  <li style="list-style-type: square">
                     <p>Type</p>
                  </li>
               </ul>
            </div><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="B">
    Are there more?  Check CSS spec.  DSSSL styling requires CSS for
    this; is there a way to do it w/o CSS?
  </p>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A" class="itemizedlist">
            <div class="title">Box marks</div>
            <ul>
               <li style="list-style-type: square">
                  <p>One</p>
               </li>
               <li style="list-style-type: square">
                  <p>Type</p>
               </li>
            </ul>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">
    Are there more?  Check CSS spec.  DSSSL styling requires CSS for
    this; is there a way to do it w/o CSS?
  </p>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>