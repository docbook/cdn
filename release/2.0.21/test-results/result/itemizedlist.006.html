<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for itemizedlist.006.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: itemizedlist.006.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">10</span> differences)</th><th>Actual (<span class="adiff">12</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: itemizedlist.006&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Unit Test: itemizedlist.006&lt;/h2&gt;</span>
            <span class="tag">&lt;div <span class="att">class="author"</span>&gt;
               <span class="tag">&lt;h3&gt;
                  <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Adam&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Di Carlo&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/h3&gt;</span>
            &lt;/div&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff del">&lt;p&gt;Itemized list mark styles.&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">itemizedlist</span>"</span>&gt;
            <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">title</span>"</span>&gt;<span class="diff txtchg">Bullet marks</span>&lt;/div&gt;</span>
            <span class="tag diff del">&lt;ul&gt;
               <span class="tag">&lt;li <span class="att">style="list-style-type: disc"</span>&gt;
                  <span class="tag">&lt;p&gt;One&lt;/p&gt;</span>
               &lt;/li&gt;</span>
               <span class="tag">&lt;li <span class="att">style="list-style-type: disc"</span>&gt;
                  <span class="tag">&lt;p&gt;Type&lt;/p&gt;</span>
               &lt;/li&gt;</span>
            &lt;/ul&gt;</span>
         &lt;/div&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;div <span class="att">class="itemizedlist"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="title"</span>&gt;Box marks&lt;/div&gt;</span>
            <span class="tag">&lt;ul&gt;
               <span class="tag">&lt;li <span class="att">style="list-style-type: square"</span>&gt;
                  <span class="tag">&lt;p&gt;One&lt;/p&gt;</span>
               &lt;/li&gt;</span>
               <span class="tag">&lt;li <span class="att">style="list-style-type: square"</span>&gt;
                  <span class="tag">&lt;p&gt;Type&lt;/p&gt;</span>
               &lt;/li&gt;</span>
            &lt;/ul&gt;</span>
         &lt;/div&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;p&gt;
    Are there more?  Check CSS spec.  DSSSL styling requires CSS for
    this; is there a way to do it w/o CSS?
  &lt;/p&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: itemizedlist.006&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Unit Test: itemizedlist.006&lt;/h2&gt;</span>
            <span class="tag">&lt;div <span class="att">class="author"</span>&gt;
               <span class="tag">&lt;h3&gt;
                  <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Adam&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Di Carlo&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/h3&gt;</span>
            &lt;/div&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">content</span>"</span>&gt;
            <span class="tag diff add">&lt;p&gt;Itemized list mark styles.&lt;/p&gt;</span><span class="diff txtchg">
            </span><span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">itemizedlist</span>"</span>&gt;<span class="diff txtchg">
               </span><span class="tag diff add">&lt;div <span class="att">class="title"</span>&gt;Bullet marks&lt;/div&gt;</span><span class="diff txtchg">
               </span><span class="tag diff add">&lt;ul&gt;
                  <span class="tag">&lt;li <span class="att">style="list-style-type: disc"</span>&gt;
                     <span class="tag">&lt;p&gt;One&lt;/p&gt;</span>
                  &lt;/li&gt;</span>
                  <span class="tag">&lt;li <span class="att">style="list-style-type: disc"</span>&gt;
                     <span class="tag">&lt;p&gt;Type&lt;/p&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span><span class="diff txtchg">
            </span>&lt;/div&gt;</span>
            <span class="tag diff add">&lt;div <span class="att">class="itemizedlist"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="title"</span>&gt;Box marks&lt;/div&gt;</span>
               <span class="tag">&lt;ul&gt;
                  <span class="tag">&lt;li <span class="att">style="list-style-type: square"</span>&gt;
                     <span class="tag">&lt;p&gt;One&lt;/p&gt;</span>
                  &lt;/li&gt;</span>
                  <span class="tag">&lt;li <span class="att">style="list-style-type: square"</span>&gt;
                     <span class="tag">&lt;p&gt;Type&lt;/p&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span><span class="diff txtchg">
            </span><span class="tag diff add">&lt;p&gt;
    Are there more?  Check CSS spec.  DSSSL styling requires CSS for
    this; is there a way to do it w/o CSS?
  &lt;/p&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: itemizedlist.006</h2><div class="author"><h3><span class="personname"><span class="firstname">Adam</span> <span class="surname">Di Carlo</span></span></h3></div></header><p>Itemized list mark styles.</p><div class="itemizedlist"><div class="title">Bullet marks</div><ul><li style="list-style-type: disc"><p>One</p></li><li style="list-style-type: disc"><p>Type</p></li></ul></div><div class="itemizedlist"><div class="title">Box marks</div><ul><li style="list-style-type: square"><p>One</p></li><li style="list-style-type: square"><p>Type</p></li></ul></div><p>
    Are there more?  Check CSS spec.  DSSSL styling requires CSS for
    this; is there a way to do it w/o CSS?
  </p></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: itemizedlist.006</h2><div class="author"><h3><span class="personname"><span class="firstname">Adam</span> <span class="surname">Di Carlo</span></span></h3></div></header><div class="content"><p>Itemized list mark styles.</p><div class="itemizedlist"><div class="title">Bullet marks</div><ul><li style="list-style-type: disc"><p>One</p></li><li style="list-style-type: disc"><p>Type</p></li></ul></div><div class="itemizedlist"><div class="title">Box marks</div><ul><li style="list-style-type: square"><p>One</p></li><li style="list-style-type: square"><p>Type</p></li></ul></div><p>
    Are there more?  Check CSS spec.  DSSSL styling requires CSS for
    this; is there a way to do it w/o CSS?
  </p></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/itemizedlist.006.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>