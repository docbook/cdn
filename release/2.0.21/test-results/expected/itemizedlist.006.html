<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: itemizedlist.006</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: itemizedlist.006</h2><div class="author"><h3><span class="personname"><span class="firstname">Adam</span> <span class="surname">Di Carlo</span></span></h3></div></header><p>Itemized list mark styles.</p><div class="itemizedlist"><div class="title">Bullet marks</div><ul><li style="list-style-type: disc"><p>One</p></li><li style="list-style-type: disc"><p>Type</p></li></ul></div><div class="itemizedlist"><div class="title">Box marks</div><ul><li style="list-style-type: square"><p>One</p></li><li style="list-style-type: square"><p>Type</p></li></ul></div><p>
    Are there more?  Check CSS spec.  DSSSL styling requires CSS for
    this; is there a way to do it w/o CSS?
  </p></article><script src="../../resources/js/prism.js"></script></body></html>